third_party/tsl/third_party/nccl/hermetic/cuda_nccl.BUILD.tpl

```diff
@@ -0,0 +1,30 @@
+licenses(["restricted"])  # NVIDIA proprietary license
+
+exports_files([
+    "version.txt",
+])
+%{multiline_comment}
+cc_import(
+    name = "nccl_shared_library",
+    shared_library = "lib/libnccl.so.%{libnccl_version}",
```

Comment:
One minor issue here will be that NCCL local install paths on Ubuntu look like this:
```
/usr/include/nccl*.h
/usr/lib/$(uname -m)-linux-gnu/libnccl*.so*
```

We could add some further symlinks to normalize this on our end so that `/usr` is the install prefix; it would be a bit messy to deal with here, I would imagine.

Reply:
It would be nice to have a consolidated approach across system-wide libraries and redistributions content. The redistributions have `lib` folder in them. We would really appreciate it if NVIDIA adds symlinks as `/usr/lib` for `/usr/lib/$(uname -m)-linux-gnu/`.

Reply:
I can live with /usr/lib as the standard -- we will adapt our build system to it using some symlinks as you said.